### ISSUE TITLE
YogPort Checklist

### ISSUE THREAD

**Issue by Super3222** [yogstation13/yogstation#4]
COPYPASTA REEEEE //Oisin was here//

#### Things we're keeping: (things that aren't checked are not finished)
- [x] The Map*
- [ ] Cybermen*
- [x] Borers*
- [x] Races*
- [x] Medical Rework*
- [x] Tickets*
- [x] Miner Drones*
- [x] Bucket heads*
- [ ] Better NTSL Tracking*
- [x] Mindslave implants*
- [x] Throwable Spells*
- [x] Floodlights *
- [x] Our changelings*
- [ ] Yogbot13*
- [ ] Yogstats13*
- [x] Shotgun ammo in armoury*&
- [x] New Emotes*
- [x] Riot Shields*
- [ ] Random sillicon laws(Yes, I know its broken anyway right now)*
- [x] Our gang updates*
- [x] Our Rev nerfs/buffs*
- [x] Detomatix not being shit*
- [ ] Pickpocket gloves*
- [x] Antag weight*
- [ ] Our deployable barrier buffs*&
- [ ] Our alternative Abductor gamemodes*
- [ ] Tactical animals*
- [ ] Syndicate scout drones?*
- [ ] Our antag selection preference fix(Not sure if /TG) has this already.*
- [ ] Telescience redesign*&
- [ ] Zombies(/TG/ Has zombies already, I prefer Yog zombies more though.)*
- [x] Wanted Poster
- [x] Re-implementing Paramedic
- [x] Re-implementing Clerk
- [x] Re-implementing Mining Medic
- [x] Our HorrorForm remake
- [x] Our Shadowling reworks, buffs, and nerfs.

- [ ] Probably a whole lot more that can be listed here. Reply in the this issue so they can be added.
*Not ported yet.
& Oisin is doing this, dont fuck with this.

#### Things we wont be keeping
Ghetto Armoury
Any /TG/ ports we've made(Obviously...)

------------
A few notes.
Please don't bombard this issue with pointless replies talking about the rebase.
If you want to get something done, than "claim" it on this issue.
If you aren't involved with the rebase than please don't post here unless you have some information that helps / supports us.

**Comment by utahclock** [comment]
Our custom jobs?

**Comment by oisin100** [comment]
Dont comment on this PR if you are porting something, Create a new issue telling us what your porting.

Also, This list isnt everything we want to port its just the important stuff, port as much as you like provided its not in the "Things we wont be keeping" list.

Before starting to port something make sure there isnt someone already porting it in the issues list.

**Comment by george99g** [comment]
The ultra preference whitelist thing we have?

**Comment by Super3222** [comment]
> The ultra preference whitelist thing we have for heads of staff?

@george99g Yes

**Comment by ShadowDeath6** [comment]
Fuck we don't have our slaughter demons on the list
